packages/libssh.rb

```diff
@@ -3,52 +3,50 @@
-  @_ver = '0.9.6'
+  @_ver = '0.10.3'
   version @_ver
   @_ver_prelastdot = @_ver.rpartition('.')[0]
   license 'LGPL-2.1'
   compatibility 'all'
   source_url "https://www.libssh.org/files/#{@_ver_prelastdot}/libssh-#{@_ver}.tar.xz"
-  source_sha256 '86bcf885bd9b80466fe0e05453c58b877df61afa8ba947a58c356d7f0fab829b'
+  source_sha256 '6e889dbe4f3eecd13a452ca868ec85525ab9c39d778519a9c141b83da738c8aa'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.6_armv7l/libssh-0.9.6-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.6_armv7l/libssh-0.9.6-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.6_i686/libssh-0.9.6-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.6_x86_64/libssh-0.9.6-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.10.3_armv7l/libssh-0.10.3-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.10.3_armv7l/libssh-0.10.3-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.10.3_i686/libssh-0.10.3-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.10.3_x86_64/libssh-0.10.3-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '5c4a6da32f3b23488dd15e06a5eb77e191143e4d5b5a396331a429e9d5dd5cfd',
-     armv7l: '5c4a6da32f3b23488dd15e06a5eb77e191143e4d5b5a396331a429e9d5dd5cfd',
-       i686: '9a3d676f230262a5974fe05e7ea705c9062a5b27364655e29e2abf4158009389',
-     x86_64: 'a4a261a53541349d5c9564a74f7a5d5051d33a4cbdbc06885810eb948f0d01a8'
+    aarch64: '6d9677de1be1914cbfa478c80b25101304ead4f467d5e8970644a53be7e287d1',
+     armv7l: '6d9677de1be1914cbfa478c80b25101304ead4f467d5e8970644a53be7e287d1',
+       i686: 'fe25cd27845a0b9c1699ffc1c9cd118744d839801ab36152e492e074f58ee6e0',
+     x86_64: 'bfed4aeabbf90f40f8a369bba7a208e89c726f991af787d4c54ed51a593685e1'
   })
 
   depends_on 'libgcrypt'
 
   def self.build
     FileUtils.mkdir('builddir')
     Dir.chdir('builddir') do
-      system "cmake #{CREW_CMAKE_OPTIONS} \
+      system "mold -run cmake #{CREW_CMAKE_OPTIONS} \
       -DWITH_EXAMPLES=OFF \
       -DBUILD_SHARED_LIBS=OFF \
-      -DWITH_STATIC_LIB=ON \
       ../ -G Ninja"
     end
-    system 'ninja -C builddir'
+    system 'mold -run samu -C builddir'
     Dir.chdir('builddir') do
       FileUtils.cp 'src/libssh.a', '../' if File.exist?('src/libssh.a')
-      system "cmake #{CREW_CMAKE_OPTIONS} \
+      system "mold -run cmake #{CREW_CMAKE_OPTIONS} \
       -DWITH_EXAMPLES=OFF \
       -DBUILD_SHARED_LIBS=ON \
-      -DWITH_STATIC_LIB=OFF \
       ../ -G Ninja"
     end
-    system 'ninja -C builddir'
+    system 'mold -run samu -C builddir'
```

Comment:
Will it make the `CREW_LINKER` constant meaningless if we use `mold -run` in every package?

Reply:
Yes.

Reply:
Yes, but this also lets us avoid hacking up everything with `-fuse-ld=mold`. It would be nice for CREW_LINKER to automatically add `mold -run` to the command line...